package.json

```diff
@@ -2,16 +2,12 @@
   "name": "@genesislcap/docs",
   "description": "Genesis platform documentation",
   "version": "0.0.1",
-  "engines": {
-    "node": ">=20.10.0"
-  },
   "scripts": {
-    "prebuild": "npm audit --parseable | grep 'critical' && echo 'Critical vulnerabilities found. Please fix them before building. To do that run npm audit.' && exit 1 || true",
     "docusaurus": "docusaurus",
     "build": "npm run copy:dsc && docusaurus build",
     "build-main": "docusaurus build",
-    "build:base_url": "cross-env BASE_URL=/docs/ npm run copy:dsc && docusaurus build",
-    "build:production": "cross-env BASE_URL=/docs/ BRANCH=master npm run copy:dsc && docusaurus build",
```

Comment:
No need to revert this feature (I'm guessing you just went back to an older version of this file)
The minimum changes for the downgrade in terms of dependencies should look something like this https://github.com/genesiscommunitysuccess/docs/pull/1563/files
(but will still need the changes to doc files to avoid the 'syntax errors')